controlled-vocabulary/src/test/java/org/uniprot/cv/xdb/UniProtKBCrossReferenceDisplayOrderTest.java

```diff
@@ -4,9 +4,11 @@
 
 import java.util.List;
 
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 import org.uniprot.core.cv.xdb.UniProtDatabaseDetail;
 
+@Disabled
```

Comment:
We restored it. https://www.ebi.ac.uk/uniprot/prod/generator/dr_ord it should be working now.